src/Mod/TechDraw/Gui/AppTechDrawGui.cpp

```diff
@@ -68,7 +68,6 @@
 
 // use a different name to CreateCommand()
 void CreateTechDrawCommands();
-void CreateTechDrawCommandsDims();

```

Comment:
How is this unused?

Reply:
Isn't this the definition of the function CreateTechDrawCommandsDims()?

Reply:
It is? But it looks to me like the function is called and thus used?

Reply:
CommandXXX.cpp files do not have a matching .h file (don't know why).